pkg/templates/aws/resources/cloudfront_distribution.yaml

```diff
@@ -69,16 +69,6 @@ properties:
         type: string
       ForwardedValues:
         type: map
-        properties:
-          QueryString:
-            type: bool
-            default_value: true
-          Cookies:
-            type: map
-            properties:
-              Forward:
-                type: string
-                default_value: none
```

Comment:
why are these getting removed?

Reply:
cant we just not set them rather than remove the ability to utilize them

Reply:
Ah, yeah, they shouldn't - was a bit overzealous. Just need to remove the default values